cmd/eksctl-anywhere/cmd/generateclusterconfig.go

```diff
@@ -203,15 +206,23 @@ func generateClusterConfig(clusterName string) error {
 				return fmt.Errorf("failed to generate cluster yaml: %v", err)
 			}
 			datacenterYaml = dcyaml
-
-			cpMachineConfig, err := v1alpha1.NewTinkerbellMachineConfigGenerate(providers.GetControlPlaneNodeName(clusterName))
+			versionBundle, err := cluster.GetVersionsBundleForVersion(version.Get(), v1alpha1.GetDefaultKubernetesVersion())
```

Comment:
What does a 'default Kubernetes version' mean in this context? I.e. What API is the default version tied to and how is that defined?

Reply:
Actually i meant for this to define the default k8s version EKS-A uses.
I can make the name more descriptive 👍